woocommerce/wc-admin
# Add your first product: email notification

This is a follow-up of [PR 5922](https://github.com/woocommerce/woocommerce-admin/pull/5922).

This PR adds a note type `email`.
This email note should be sent:
- after three days.
- if the user doesn't have any products.

### Accessibility




### Screenshots
![screenshot-one wordpress test-2021 01 08-11_52_25](https://user-images.githubusercontent.com/1314156/104028917-10043980-51a8-11eb-946f-0ac36d8ba69d.png)


### Detailed test instructions:

- Checkout this branch.
- Create a zip for testing with `npm run zip:test`.
- Create a `jurassic.ninja` instance.
- Upload the plugin and activate it.
- Update the installation date (we need a 3-day old store). You can do it with an SQL sentence like this:

```
UPDATE `wp_options` SET `option_value`=UNIX_TIMESTAMP(DATE_SUB(NOW(), INTERVAL 30 day)) WHERE `option_name` = 'woocommerce_admin_install_timestamp';
```

- Run the cron (this tool can help [WP Crontrol](https://wordpress.org/plugins/wp-crontrol/)).
- You should have received an email like the image above.
- Verify the note's status is `sent`. You can use a sentence like this:

```
SELECT `status` FROM `wp_wc_admin_notes` WHERE `name` = 'wc-admin-add-first-product-note'
```

- Go back to the email and click `Add a product`.
- It should take you to the product page.
- Now the note's status should be `actioned`.
